dev/TeachingTip/TeachingTip.cpp

```diff
@@ -119,6 +126,17 @@ void TeachingTip::OnPropertyChanged(const winrt::DependencyPropertyChangedEventA
     }
     else if (property == s_TargetProperty)
     {
+        // Unregister from old target if it exists
+        if (args.OldValue()) {
+            winrt::FrameworkElement oldTarget = unbox_value<winrt::FrameworkElement >(args.OldValue());
+            m_TargetUnloadedRevoker.revoke();
+        }
+
+        // Register to new target if it exists
+        if (args.NewValue()) {
```

Comment:
>args.NewValue() [](start = 12, length = 15)

This is slightly better I think:
`
if(const auto& value = args.NewValue()) {
                winrt::FrameworkElement newTarget = unbox_value<winrt::FrameworkElement >(value));
            m_TargetUnloadedRevoker = newTarget.Unloaded(winrt::auto_revoke,{ this,&TeachingTip::ClosePopupOnUnloadEvent });
}
`

Reply:
Great idea. Fixed that now :)